StringCalculator.Core/Processor/StringCalculatorProcessor.cs

```diff
@@ -0,0 +1,58 @@
+﻿using System;
+using System.Linq;
+
+namespace StringCalculator.Core.Processor
+{
+    public  class StringCalculatorProcessor
+    {
+        public StringCalculatorProcessor()
+        {
+        }
+
+        public  int Add(string numbers)
+        {
+            if (numbers.Equals(""))
```

Comment:
let's start using C# defined functions that are easier and more readable, for example, here you can use string.IsNullOrEmpty(numbers) function instead of compare with ""